### PR TITLE
Fix loading of local config in Docker

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A  |
|---          |---|
|Fixed tickets|   |
|License      |MIT|
                   

For real this time 😅

### CODE REVIEW

File: src/ConfigFactory.php [hunk 1]
```diff
@@ -110,6 +110,7 @@ public static function createConfig($localHome = null)
     public static function createConfigFromEnv($systemConfigEnv = null, $localConfigEnv = null)
     {
         $cacheDir = '/tmp';
```
Comment (sstok) [inline src/ConfigFactory.php]: Properly would be better to remove caching all together in a follow-up pr.
This is only used by the GitHub adapter atm.

Comment (phansys) [inline src/ConfigFactory.php]: It could be nice to use `sys_get_temp_dir()` in order to be cross platform.

Comment (sstok) [inline src/ConfigFactory.php]: `createConfigFromEnv()` is actually only used by Docker :)

Comment (cordoval) [inline src/ConfigFactory.php]: I have been removing caching things for a while now, it is an example of a bad decision gone rogue 😊

**Comment by cordoval** [comment]
@sstok could you rebase please?

File: gush.dist [hunk 2]
```diff
@@ -3,5 +3,7 @@
 docker run -it \
     -v ~/.ssh/id_rsa.pub:/root/.ssh/id_rsa.pub \
     -v ~/.ssh/id_rsa:/root/.ssh/id_rsa \
-    -e GUSH_CONFIG="$(cat ~/.gush/.gush.yml | base64)" \
-    coder20078/gush "$@"
+    -v ~/.ssh/known_hosts:/root/.ssh/known_hosts \
+    -e GUSH_CONFIG=`cat ~/.gush/.gush.yml | base64` \
+    -e GUSH_LOCAL_CONFIG=`cat $(pwd)/.gush.yml | base64` \
+	coder20078/gush "$@"
```
Comment (sstok) [inline gush.dist]: whoops wrong indent

**Pull request merged**